_posts/doc/2016-07-16-Write Custom Theme.md

```diff
@@ -45,11 +45,11 @@ The variables may change in future, so you [could use DevTools in Typora to conf
 
 If you want to write a custom CSS theme for Typora, all you need to do is:
 
-1. New a css file. The file name **should not include capitalised characters or whitespace**, for example: `my-typora-theme` is a valid file name.
+1. Create a new css file. The file name **should not include capitalised characters or whitespace**, for example: `my-typora-theme` is a valid file name.
 
 2. Write the css file. 
 
-   We prepare a [toolkit][] for you to get start or do simple test. 
+   We prepared a [toolkit] for you to get started or to do simple testing. 
```

Comment:
`[toolkit]` is not supported by Typora yet, please still use `[toolkit][]` instead.

Reply:
I thought this was supposed to link to a page?!?

Reply:
In Markdown, `[toolkit][]` is equal to `[toolkit][toolkit]`, which links to a reference link, defined by `[toolkit]: http://aaa.com`.